testsuite/single_node_performance.py

```diff
@@ -156,6 +156,8 @@ class RunGroupConfig:
     RunGroupConfig(expected_tps=970, key=RunGroupKey("liquidity-pool-swap-stable"), included_in=Flow.CONTINUOUS),
     RunGroupConfig(expected_tps=4350, key=RunGroupKey("liquidity-pool-swap-stable", module_working_set_size=20), included_in=Flow.CONTINUOUS),
 
+    RunGroupConfig(expected_tps=27000, key=RunGroupKey("deserialize-u256"), included_in=LAND_BLOCKING_AND_C),
```

Comment:
let's make it continuous, instead of land blocking

Reply:
Done.